src/application/Import.java

```diff
@@ -49,8 +59,43 @@ private boolean is_zip_check(String name){
         return false;
     }
 
-    public boolean isValidPackage(String path) {
-        return false;
+    public boolean isValidPackage(String name) throws IOException {
+        packagePath="data/"+name;
+        ArrayList<Lesson> pom = GetLessonsFromImportedXML(packagePath);
+        System.out.println(pom);
+        Iterator<Lesson> it = pom.iterator();
```

Comment:
nebolo by kusok krajsie kebyze tu je namiesto 'it', 'iter', a 'i' : 'lessonIterator', 'groupIterator' a 'itemIterator'?
Kebyze to ma este 3 vnorene iteratory tak sa volaju 'ite', 'itera' a 'iterat' ? :D :)